demo/kubernetes/rook-cluster.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 spec:
   versionTag: master-latest
   dataDirHostPath:
+  placement:
```

Comment:
could you fill in this example structure with each of the daemons and `all`? And it would be helpful to include a sample toleration and nodeAffinity, even if they remain commented out.